ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/PackageManagementServiceIT.scala

```diff
@@ -45,6 +45,62 @@ final class PackageManagementServiceIT extends LedgerTestSuite {
     }
   })
 
+  test(
+    "DuplicateSubmissionIdTwiceCorrect",
+    "Uploading a package twice with the same submissionId succeeds only on two participants",
+    allocate(NoParties, NoParties),
+  )(implicit ec => { case Participants(Participant(alpha), Participant(beta)) =>
+    // Multiple package updates should always succeed. Participant adds extra entropy to the
+    // submission id to ensure client does not inadvertently cause problems by poor selection
+    // of submission ids.
+    for {
+      testPackage <- loadTestPackage()
+      request = alpha.uploadDarRequest(testPackage)
+      _ <- alpha.uploadDarFile(request)
+      _ <- beta.uploadDarFile(request)
+    } yield ()
+  })
+
+  test(
+    "DuplicateSubmissionIdFirstIncorrect",
+    "Uploading package twice with the same submissionId fails for first incorrect submission",
+    allocate(NoParties, NoParties),
+  )(implicit ec => { case Participants(Participant(alpha), Participant(beta)) =>
+    for {
+      testPackage <- loadTestPackage()
+      goodRequest = alpha.uploadDarRequest(testPackage)
+      badRequest = goodRequest.update(_.darFile := ByteString.EMPTY)
+      failure <- alpha.uploadDarFile(badRequest).mustFail("uploading an empty package")
+      _ <- beta.uploadDarFile(goodRequest)
+    } yield {
+      assertGrpcError(
+        failure,
+        Status.Code.INVALID_ARGUMENT,
+        "Invalid argument: Invalid DAR: package-upload",
+      )
+    }
+  })
+
+  test(
+    "DuplicateSubmissionIdSecondIncorrect",
+    "Uploading package twice with the same submissionId fails for second incorrect submission",
```

Comment:
The same comment as for the `ConfigManagementServiceIT` wrt to test descriptions and similarity.

Reply:
Done